src/sidebars/search/search-controller.js

```diff
@@ -21,7 +21,12 @@
         }
 
         select(stock) {
-            this.selectionService.select(stock);
+            if (!this.store) {
+                this.store = window.storeService.open(window.name);
+            }
+            if (!this.store.isCompact() && stock.favourite) {
+                this.selectionService.select(stock);
+            }
```

Comment:
>and in full view search results that aren't favourites already can't be selected

In full view everything in the search tab can be selected, favourites or search results alike.

The way I understood it would work was for the search tab:

Full view - everything can be selected
Compact view - nothing can be selected

Though I might have misunderstood something.